setup-mac.py

```diff
@@ -0,0 +1,60 @@
+"""
+Setup file for mimic
+"""
+
+from setuptools import setup, find_packages
+from twisted.plugin import getPlugins, IPlugin
+from py2app.build_app import py2app
+from mimic import plugins
+
+
+NAME = 'mimic'
+VERSION = '0.1'
+ID = 'mimic'
+SCRIPT='bundle/start-app.py'
+TEST_SCRIPT='bundle/run-tests.py'
+PLIST = dict(
+    CFBundleName                = NAME,
+    CFBundleShortVersionString  = ' '.join([NAME, VERSION]),
+    CFBundleGetInfoString       = NAME,
+    CFBundleExecutable          = NAME,
+    CFBundleIdentifier          = 'com.yourdn.%s' % ID,
+    LSUIElement                 = '1',
+    LSMultipleInstancesProhibited = '1',
```

Comment:
Interesting. I guess we have to prohibit multiple instances because we have no mechanism for customizing the port number or logfile location?